cmd/auth/main.go

```diff
@@ -135,7 +133,6 @@ func main() {
 		ReservedNameChecker:      reservedNameChecker,
 	}
 
-	task.AttachVerifyCodeSendTask(asyncTaskExecutor, authDependency)
```

Comment:
Remove the configuration as well.

Reply:
I planned to cleanup & refactor the configuration (config/tenant.go) as a separate task.